rsts/index.rst

```diff
@@ -54,27 +54,208 @@ Created at `Lyft <https://www.lyft.com/>`__ in collaboration with Spotify, Freen
 
 The core unit of execution in Flyte is the ``task``, which you can easily write with the Flytekit Python SDK:
 
-.. code:: python
-   
-   @task
-   def greet(name: str) -> str:
-       return f"Welcome, {name}!"
+.. tabs::
 
-You can compose one or more tasks to create a ``workflow``:
+    .. tab:: python
```

Comment:
Is there an obvious link in the page that takes you to the rest of the python examples? if not, let's add one, if yes, then let's add a link to the "java examples page" (if that's a thing) as well...

Or maybe they should be at the end of teach of the tabs?

Reply:
I don't think there is.